client.js

```diff
@@ -87,6 +87,14 @@ function Client (peerId, port, torrent, opts) {
       } else if (protocol === 'udp:' && typeof UDPTracker === 'function') {
         return new UDPTracker(self, announceUrl)
       } else if ((protocol === 'ws:' || protocol === 'wss:') && webrtcSupport) {
+        // Don't try to add http tracker on an https website
+        if (protocol === 'ws:' && location && location.protocol && location.protocol === 'https:') {
```

Comment:
This will crash in Node.js (for example, in `webtorrent-hybrid`) because `location` is not defined.